pmd-cataloger/src/test/java/sfdc/sfdx/scanner/pmd/PmdRuleCatalogerTest.java

```diff
@@ -152,10 +146,13 @@ public void testExceptionIsThrownWhenCollisionOccurs() {
 				COLLISION_JAR_2.toAbsolutePath().toString()));
 		PmdRuleCataloger pmdRuleCataloger = new PmdRuleCataloger(rulePathEntries);
 
-		thrown.expect(new MessagePassableExceptionMatcher(EventKey.ERROR_EXTERNAL_DUPLICATE_XML_PATH,
-				new String[] { "category/joshapex/somecat.xml", COLLISION_JAR_2.toAbsolutePath().toString(),
-						COLLISION_JAR_1.toAbsolutePath().toString() }));
-
-		pmdRuleCataloger.catalogRules();
+        MessagePassableException ex = assertThrows(MessagePassableException.class,
```

Comment:
We no longer need MessagePassableExceptionMatcher since i've converted all these thrown.expect calls into assertThrows with additional assertions on the expected message.